R/textplot_wordcloud.R

```diff
@@ -1,70 +1,468 @@
-#' document.
+#'
+#' Plot a \link{dfm} object as a wordcloud, where the feature labels are plotted
+#' with their sizes proportional to their numerical values in the dfm.  When
+#' \code{comparison = TRUE}, it plots comparison word clouds by document.
 #' @details The default is to plot the word cloud of all features, summed across
-#'   documents.  To produce word cloud plots for specific document or set of 
-#'   documents, you need to slice out the document(s) from the dfm or tokens
-#'   object.
+#'   documents.  To produce word cloud plots for specific document or set of
+#'   documents, you need to slice out the document(s) from the dfm object.
 #'   
 #'   Comparison wordcloud plots may be plotted by setting \code{comparison =
 #'   TRUE}, which plots a separate grouping for \emph{each document} in the dfm.
 #'   This means that you will need to slice out just a few documents from the
 #'   dfm, or to create a dfm where the "documents" represent a subset or a
 #'   grouping of documents by some document variable.
+#'   
 #' @param x a dfm object
-#' @param comparison if \code{TRUE}, plot a 
-#'   \code{\link[wordcloud]{comparison.cloud}} instead of a simple wordcloud, 
-#'   one grouping per document
-#' @param ... additional parameters passed to \link[wordcloud]{wordcloud} or 
-#'   to \link{text} (and \link{strheight}, \link{strwidth})
-#' @seealso \code{\link[wordcloud]{wordcloud}}, 
-#'   \code{\link[wordcloud]{comparison.cloud}}
+#' @param min_size size of the smallest word
+#' @param max_size size of the largest word
+#' @param max_words maximum number of words to be plotted. least frequent terms
+#'   dropped.
+#' @param color color of words from least to most frequent
+#' @param font font-family of words and labels. Use default font if \code{NULL}.
+#' @param adjust ajust sizes of words by a constant. Useful for non-Engish words
+#'   for which R fails to obtain correct sizes.
+#' @param rotation proportion of words with 90 degree rotation
+#' @param random_order plot words in random order. If \code{FALSE}, they will be
+#'   plotted in decreasing frequency.
+#' @param random_color choose colors randomly from the colors. If \code{FALSE},
+#'   the color is chosen based on the frequency
+#' @param ordered_color if \code{TRUE}, then colors are assigned to words in
+#'   order.
+#' @param labelcolor color of group labels. Only used when \code{compariosn=TRUE}.
+#' @param labelsize size of group labels. Only used when \code{compariosn=TRUE}.
+#' @param labeloffset  position of group labels. Only used when
+#'   \code{comparison=TRUE}.
+#' @param fixed_aspect if \code{TRUE}, the aspect ratio is fixed. Variable
+#'   aspect ratio only supported if rotation = 0
+#' @param comparison if \code{TRUE}, plot a wordclound that compares documents
+#'   in the same way as \code{\link[wordcloud]{comparison.cloud}}
+#' @param ... additional parameters passed to \link{text} (and \link{strheight},
```

Comment:
This is indeterminate, since it passes any arguments not in the list or the deprecated set through to `graphics::strwidth()` and `graphics::strwidth()` and `text()`. The parameters for the first two functions are different from those for the third.

Reply:
Stopped passing ... to graphic functions.